### PR TITLE
Feature/ong view get

### PR DESCRIPTION
**Pull request by pronguto**

### CODE REVIEW

File: ongs/urls.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from .views import OngIdView, OngView
+
+urlpatterns = [
+    path("ongs/", OngView.as_view()),
+    path("ongs/<int:ong_id>", OngIdView.as_view()),
```
Comment (marcelosj3) [inline ongs/urls.py:7]: missing `/` at the end of the url: `"ongs/<int:ong_id>/"`

File: ongs/views.py [hunk 2]
```diff
@@ -1,3 +1,27 @@
-from django.shortcuts import render
+from django.http import Http404
+from django.shortcuts import get_object_or_404, render
+from rest_framework.views import APIView, Request, Response, status
+from yaml import serialize
```
Comment (marcelosj3) [inline ongs/views.py:4]: Is this import correct?

File: sos_brazil/urls.py [hunk 3]
```diff
@@ -14,8 +14,6 @@
     2. Add a URL to urlpatterns:  path('blog/', include('blog.urls'))
 """
 from django.contrib import admin
-from django.urls import path
+from django.urls import include, path
 
-urlpatterns = [
-    path("admin/", admin.site.urls),
-]
+urlpatterns = [path("admin/", admin.site.urls), path("api", include("ongs.urls"))]
```
Comment (marcelosj3) [inline sos_brazil/urls.py:19]: missing `/` in the api url

**Review by marcelosj3 — CHANGES_REQUESTED** [review]

File: ongs/urls.py (hunk 1 above)
Comment (pronguto) [reply ongs/urls.py]: Done!

File: ongs/views.py (hunk 2 above)
Comment (pronguto) [reply ongs/views.py]: No, I remove the unused imports.

**Review by marcelosj3 — APPROVED** [review]

**Review by gabrielricardo7 — APPROVED** [review]

**Pull request merged**